src/main/java/edu/cornell/kfs/sys/xmladapters/RassStringToJavaLocalDateAdapter.java

```diff
@@ -0,0 +1,35 @@
+package edu.cornell.kfs.sys.xmladapters;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+import jakarta.xml.bind.annotation.adapters.XmlAdapter;
+
+import org.apache.commons.lang3.StringUtils;
+
+import edu.cornell.kfs.sys.CUKFSConstants;
+
+public class RassStringToJavaLocalDateAdapter extends XmlAdapter<String, LocalDate> {
+
+    protected static final DateTimeFormatter DATE_LOCAL_DATE_FORMATTER_yyyy_MM_dd = 
+            DateTimeFormatter.ofPattern(CUKFSConstants.DATE_FORMAT_yyyy_MM_dd).withLocale(Locale.US);
+
+    @Override
+    public LocalDate unmarshal(String dateStringToConvert) throws Exception {
+        return StringUtils.isNotBlank(dateStringToConvert) ? parseToLocalDate(dateStringToConvert) : null;
+    }
+
+    @Override
+    public String marshal(LocalDate dateAsLocalDate) throws Exception {
+        return (dateAsLocalDate != null) ? dateAsLocalDate.format(DATE_LOCAL_DATE_FORMATTER_yyyy_MM_dd) : null;
```

Comment:
would it be better to return an empty String rather than a null String of dateAsLocalDate?
would the input variable be clearer if it was just localDate?

Reply:
Lets leave it as you coded it here, since this is how it worked before, should leave it the same,